playbooks/operationalizing/cloudforms.adoc

```diff
@@ -256,6 +256,109 @@ The following steps occurs during a SmartState container image analysis:
 5. CloudForms will communicate via the OpenShift API via the pod proxy to the WebDAV server to retrieve the results of the inspection 
 
 
+=== OpenSCAP Default Condition
+CloudForms Control allows you to handle OpenSCAP features such as Policy. By default, there is one condition for container image. If OpenSCAP rule result has high severity,
+it fails to deploy any container using the image.
+
+.Condition Expression
+--
+FIND Container Image.Openscap Rule Results : Result = "fail" CHECK ANY Severity = "High" 
+--
+
+image::images/cloudforms-openscap-condition.png[title=CloudForms OpenSCAP Default Condition]
+
+
+== Chargeback for OpenShift Container Platform
+Red Hat CloudForms provides Chargeback feature allows an administrator to calculate monetary projects/containers charges. The administrator creates Chargeback rate based on mainly Compute or Storage.  In order to get Chargeback report, OpenShift has to deploy metrics and Cloudforms have to be integrated with OpenShift. Moreover, it takes a couple of hours to gather data hence, it does not show any record during the time. Refer official document for link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.2/html-single/monitoring_alerts_and_reporting/#sect_chargeback[Chargeback]
```

Comment:
The requirements for metrics to be deployed along with the integration of CFME with OCP was described earlier. We can probably omit the callout for this requirement